### PR TITLE
Add Travis config

### PR DESCRIPTION
**Pull request by vidmantas**
Here's a basic Travis config :) Let me know if list of rubies are too extensive or too small in your opinion. 

When `failure_message` method is fixed, it is possible to add rubocop to the build as well ```</idea>``` ;)

### CODE REVIEW

**Comment by isaacseymour** [comment]
Fixed that cop on master, and tightened the rules a bit for fun. Thanks for this!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.2.3
+  - 2.0.0
```
Comment (isaacseymour) [inline .travis.yml]: Should probably have 2.1.x represented in here too

Comment (greysteil) [inline .travis.yml]: Also, you want to specify the minor version, not the teeny one. Ruby is SemVer these days.

**Comment by isaacseymour** [comment]
Added `2.1.7` and enabled `rubocop` on master. Build status is in the readme too :dancer:

**Pull request merged**